src/ui/React/LogBoxManager.tsx

```diff
@@ -128,6 +129,23 @@ function LogWindow(props: IProps): React.ReactElement {
     setRerender((old) => !old);
   }
 
+  useEffect(
+    () =>
+      WorkerScriptStartStopEventEmitter.subscribe(() => {
+        setTimeout(() => {
```

Comment:
Why the timeout?

Reply:
for some reason without it it didnt update i think it might be a race condition between the script starting up and being found as a "running script"